contracts/interfaces/draft-IERC6909.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (interfaces/draft-IERC6909.sol)
```

Comment:
This gets automatically added [here](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/502e1a0fe5460d5cf9c8ff5bd215a5a50180d730/scripts/release/update-comment.js), when the package gets versioned.

Let's remove it

```suggestion
```